scipy/stats/_morestats.py

```diff
@@ -3932,22 +3935,22 @@ def directionalmean(samples, *, axis=0, normalize=True):
 
     See also
     --------
-    circmean: directional mean in two dimensions for circular data
+    circmean: circular mean; i.e. directional mean for 2D *angles*
 
     Notes
     -----
     This uses a definition of directional mean from [1]_.
-    Essentially, the calculation is as follows.
+    Assuming the observations are unit vectors, the calculation is as follows.
 
     .. code-block:: python
 
         mean=samples.mean(axis=0)
         directionalmean = mean/np.linalg.norm(mean)
 
-    This definition is applicable for *directional* data but not
-    for *axial* data. In practice, this means that if your data
-    are symmetric around the origin (`x` has the same meaning as
-    -`x`), this function should not be used.
+    This definition is appropriate for *directional* data (i.e. vector data
+    for which the magnitude of each observation is irrelevant) but not
+    for *axial* data (i.e. vector data for which the magnitude and *sign* of
```

Comment:
"data are symmetric around the origin" does not really mean the same thing as "`x` has the same meaning as
    -`x`".

Would you agree with this way of expressing the difference between "axial" and "directional" data?

Reply:
Yes, that definition is correct.